modules/addNavigation.js

```diff
@@ -0,0 +1,37 @@
+/**
+ * History enhancer that adds additional navigation methods — `transitionTo()`
+ * and `replaceWith()`. TODO: move to history module (?)
+ * @param {CreateHistory} createHistory - History-creating function
+ * @returns {CreateHistory}
+ */
+export default function addNavigation(createHistory) {
+  return (...args) => {
+    const history = createHistory(...args);
```

Comment:
I think we can

```js
const history = enableQueries(createHistory)(...args);
```

here just so we can be sure that people are using a query-enabled history, right? I'll probably need to update `enableQueries` to take its `options` just like `createHistory` does (as an arg to the function it returns).